Arrowgene.Ddon.GameServer/Characters/ExpManager.cs

```diff
@@ -333,14 +333,16 @@ public class ExpManager
                 },
             };
 
-        public ExpManager(DdonServer<GameClient> server, GameClientLookup gameClientLookup)
+        public ExpManager(DdonServer<GameClient> server, GameClientLookup gameClientLookup, GameLogicSetting gameSettings)
```

Comment:
We could even save the extra arg if we change the first from DdonServer<GameClient> to DdonGameServer and access the settings through this._Server.Setting